runtime/benches/accounts.rs

```diff
@@ -0,0 +1,73 @@
+#![feature(test)]
+
+extern crate test;
+
+use solana_runtime::bank::*;
+use solana_sdk::account::Account;
+use solana_sdk::genesis_block::GenesisBlock;
+use solana_sdk::pubkey::Pubkey;
+use solana_sdk::signature::{Keypair, KeypairUtil};
+use std::sync::Arc;
+use test::Bencher;
+
+fn compare_account(account1: &Account, account2: &Account) -> bool {
+    if account1.data != account2.data
+        || account1.owner != account2.owner
+        || account1.executable != account2.executable
+        || account1.lamports != account2.lamports
+    {
+        return false;
+    }
+    true
+}
+
+fn create_account(bank: &Bank, pubkeys: &mut Vec<Pubkey>, num: usize) {
```

Comment:
Typically things called `create_account` or `new_account` create one account using lamports from another.  This one creates lamports out of thin air for multiple accounts.  Can you give it a name that reflects that?  Maybe something like `deposit_many`?